improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -256,11 +399,47 @@ def test_values(self):
         assert_array_equal(threshold_0, expected_0)
         assert_array_equal(threshold_1, expected_1)
 
+    def test_combine_undersampled_bins(self):
+        """Test expected values are returned when an under-sampled bin is
+        combined with its neighbour."""
+
+        reliability_cube = self.reliability_cube[0].copy()
+        reliability_cube.data = np.array(
+            [
+                [0, 250, 50, 750, 1000],  # Observation count
+                [0, 250, 50, 750, 1000],  # Sum of forecast probability
+                [1000, 1000, 100, 1000, 1000],  # Forecast count
+            ],
+            dtype=np.float32,
+        )
+
+        result = self.plugin._calculate_reliability_probabilities(reliability_cube)
+
+        self.assertIsNone(result[0])
```

Comment:
This test is currently a placeholder until #1315 is merged. The commented out test below, which shows under-sampled bins being combined and non-None values being computed for the forecast probability and observed frequency, is preferable.